sway/desktop/fx_renderer/shaders/box_shadow.frag

```diff
@@ -65,7 +66,8 @@ void main() {
             position + blur_sigma,
             position + size - blur_sigma,
             gl_FragCoord.xy, blur_sigma * 0.5,
-            corner_radius);
+            corner_radius,
+            offset);
```

Comment:
I'd adjust the `lower` and `upper` parameters instead of introducing a new offset parameter

Reply:
Yup, could probably be done by changing the position input instead :)

Reply:
It already works without the offset because I am adjusting the box area on which the blur gets applied, so I could remove the offset from the shader entirely.